src/main/scala/codacy/codesniffer/docsgen/parsers/DocsParser.scala

```diff
@@ -0,0 +1,52 @@
+package codacy.codesniffer.docsgen.parsers
+
+import java.nio.file.Files
+
+import better.files.File
+import com.codacy.plugins.api.results.{Parameter, Pattern, Result}
+import com.codacy.tools.scala.seed.utils.CommandRunner
+
+case class PatternDocs(pattern: Pattern.Specification, description: Pattern.Description, docs: Option[String])
+
+trait DocsParser {
+  def repositoryURL: String
+
+  def handleRepo(dir: File): Set[PatternDocs]
+
+  def patterns: Set[PatternDocs] =
+    withRepo(repositoryURL)(handleRepo)
+      .fold(a => throw a, identity)
+
+  private[this] def withRepo[A](repositoryURL: String)(f: File => A): Either[Throwable, A] = {
+    val dir = Files.createTempDirectory("")
+    for {
+      _ <- CommandRunner
+        .exec(List("git", "clone", repositoryURL, dir.toString))
```

Comment:
Always clone a specific version matching the one used to build the docker.

Reply:
Fixed